media/cdm/BUILD.gn

```diff
@@ -170,6 +170,21 @@ source_set("cdm") {
     public_deps += [ "//third_party/fuchsia-sdk/sdk/fidl/fuchsia.media.drm:fuchsia.media.drm_hlcpp" ]
     deps += [ "//media/fuchsia/common" ]
   }
+
+  if (is_cobalt) {
+    import("//starboard/build/buildflags.gni")
+    sources += [
+       "starboard/starboard_cdm.cc",
```

Comment:
We should add:
- `/media/cdm/starboard/ # no owners` to `.github/CODEOWNERS`
- `media/cdm/starboard/|` to `.pre-commit-config.yaml`

So the starboard codes can get formatted.